opentreemap/importer/views.py

```diff
@@ -110,6 +113,18 @@ def get_import_table(request, instance, table_name):
 _EVENT_TABLE_PAGE_SIZE = 5
 
 
+def _cleanup_tables(instance):
+    ievents = (list(TreeImportEvent.objects.filter(instance=instance)) +
+               list(SpeciesImportEvent.objects.filter(instance=instance)))
+    for ie in ievents:
+        if ie.task_id != '':
+            result = AsyncResult(ie.task_id)
+            if result.failed():
```

Comment:
Probably better to be conservative about assigning this error state in fear of accidentally applying it to a perfectly valid job. If being unable to retrieve the `AsyncResult` turns out to be a problem in the wild, we can address it then. Does that make sense?